modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/js/ckeditor5/ClassicEditor.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {CKEditor} from '@ckeditor/ckeditor5-react';
+import {
+	Bold,
+	ClassicEditor as BaseClassicEditor,
+	EditorConfig,
+	Essentials,
+	GeneralHtmlSupport,
+	Italic,
+	Paragraph,
+	Style,
+	StyleConfig,
+	Underline,
+} from 'ckeditor5';
+import React from 'react';
+
+import Timestamp from './plugins/Timestamp';
+
+interface ICKEditor5EditorConfig extends EditorConfig {
+	editorTransformerURLs?: Array<string>;
+}
+
+const ClassicEditor = ({config}: {config?: ICKEditor5EditorConfig}) => {
+	const styleConfig: StyleConfig = {
+		definitions: [
+			{
+				classes: [],
+				element: 'p',
+				name: 'Normal',
+			},
+			{
+				classes: [],
+				element: 'h1',
+				name: 'Heading 1',
+			},
+		],
+	};
```

Comment:
Many plugins have their own configuration within the main configuration, just like in CKE4.

In my opinion, we should set all defaults here, in JS. Defaults should not be in base EditorConfigContributors, like [style plugin config for CKE4](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/java/com/liferay/frontend/editor/ckeditor/web/internal/editor/configuration/CKEditorConfigContributor.java#L131-L157). If we set them here, we promote easier frontend-only usage.